### PR TITLE
fix(sidebar): add light/dark theme border, remove desktop shadow

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/4925

### CODE REVIEW

**Comment by mmenestr** [comment]
Oh we're getting rid of the shadow for the light theme nav as well?

**Review by mmenestr — APPROVED** [review]
I commented something but just deleted it so ignore that haha this LGTM!

File: src/patternfly/components/Page/page.scss [hunk 1]
```diff
@@ -74,7 +74,11 @@ $pf-c-page--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg"
   --pf-c-page__sidebar--Width: #{pf-size-prem(290px)};
   --pf-c-page__sidebar--BackgroundColor: var(--pf-global--BackgroundColor--dark-300);
   --pf-c-page__sidebar--m-light--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
-  --pf-c-page__sidebar--BoxShadow: var(--pf-global--BoxShadow--lg-right);
+  --pf-c-page__sidebar--BoxShadow: none;
```
Comment (mattnolting) [inline src/patternfly/components/Page/page.scss:77]: I think want to keep this for < `$pf-global--breakpoint--xl`. You lost your `box-shadow` when the sidebar appears above the page content.

Screen Shot 2022-07-07 at 1 29 46 PM

**Comment by mcoker** [comment]
Updated for light and dark theme to have a right border, removed the shadow on desktop (> 1200px viewport) for all variations.

**Review by mattnolting — APPROVED** [review]
LPTM 👍

**Review by mceledonia — APPROVED** [review]

**Review by mmenestr — APPROVED** [review]

**Pull request merged**